pkg/service/model_build_target_group.go

```diff
@@ -484,6 +484,8 @@ func (t *defaultModelBuildTask) buildTargetGroupBindingNetworking(_ context.Cont
 				Protocol: &protocolTCP,
 				Port:     &tgPort,
 			})
+		case corev1.Protocol("TCP_UDP"):
+			fallthrough
```

Comment:
Do we not need to build a port-list with both TCP and UDP here? `buildTargetGroupBindingNetworkingLegacy` does it that way, but I admit I'm not familiar with the difference in these two functions.

I also note that the test for this function has not had a `TCP_UDP` test added, while `buildTargetGroupBindingNetworkingLegacy`'s test has been updated.